static/js/containers/ProfileFormContainer.js

```diff
@@ -177,6 +111,28 @@ class ProfileFormContainer extends React.Component {
     }
   }
 
+  createActionHelper: Function = (actionCreator: Function): (...args: any) => void  => {
+    const { dispatch } = this.props;
+    return (...args) => dispatch(actionCreator(...args));
+  };
```

Comment:
another question is if it makes sense to put this in another file, so that we could re-use a similar pattern on other container components (same for the method below). then we'd just do:

```js
import { createActionHelper } from '../util/redux';

createActionHelper.call(this, someAction);
```